modules/microshift-http-proxy.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: CONCEPT
+[id="microshift-http-proxy_{context}"]
+
+== Deploying {product-title} behind an HTTP(S) proxy
```

Comment:
Guidelines say not to have any spaces between id and title line (idk why, I am guessing it might break something :sweat_smile:)

```suggestion
[id="microshift-http-proxy_{context}"]
== Deploying {product-title} behind an HTTP(S) proxy
```